src/app/pages/fund/components/fiat-providers.utils.ts

```diff
@@ -0,0 +1,98 @@
+import CryptoComIcon from '@assets/images/fund/fiat-providers/crypto.com-icon.png';
+import GateIoIcon from '@assets/images/fund/fiat-providers/gate.io-icon.png';
+import KuCoinIcon from '@assets/images/fund/fiat-providers/kucoin-icon.png';
+import MoonPayIcon from '@assets/images/fund/fiat-providers/moonpay-icon.png';
+import OkcoinIcon from '@assets/images/fund/fiat-providers/okcoin-icon.png';
+import OkxIcon from '@assets/images/fund/fiat-providers/okx-icon.png';
+import TransakIcon from '@assets/images/fund/fiat-providers/transak-icon.png';
+import {
+  MOONPAY_API_KEY_PRODUCTION,
+  TRANSAK_API_KEY_PRODUCTION,
+  TRANSAK_API_KEY_STAGING,
+} from '@shared/constants';
+
+// Keys are set in wallet-config.json
+enum ActiveFiatProviders {
+  Binance = 'binance',
+  BlockchainCom = 'blockchainCom',
+  ByBit = 'byBit',
+  Coinbase = 'coinbase',
+  CryptoCom = 'cryptoCom',
+  GateIo = 'gateIo',
+  KuCoin = 'kuCoin',
+  MoonPay = 'moonPay',
+  Okcoin = 'okcoin',
+  Okx = 'okx',
+  Transak = 'transak',
+}
+
+export const activeFiatProviderIcons: Record<string, string> = {
+  [ActiveFiatProviders.Binance]: BinanceIcon,
+  [ActiveFiatProviders.BlockchainCom]: BlockchainComIcon,
+  [ActiveFiatProviders.ByBit]: ByBitIcon,
+  [ActiveFiatProviders.Coinbase]: CoinbaseIcon,
+  [ActiveFiatProviders.CryptoCom]: CryptoComIcon,
+  [ActiveFiatProviders.GateIo]: GateIoIcon,
+  [ActiveFiatProviders.KuCoin]: KuCoinIcon,
+  [ActiveFiatProviders.MoonPay]: MoonPayIcon,
+  [ActiveFiatProviders.Okcoin]: OkcoinIcon,
+  [ActiveFiatProviders.Okx]: OkxIcon,
+  [ActiveFiatProviders.Transak]: TransakIcon,
+};
+
+const isProduction =
+  process.env.WALLET_ENVIRONMENT === 'production' || process.env.WALLET_ENVIRONMENT === 'preview';
+
+function makeMoonPayUrl(address: string) {
+  return `https://buy.moonpay.com?apiKey=${MOONPAY_API_KEY_PRODUCTION}&currencyCode=stx&walletAddress=${address}`;
+}
+
+function makeOkcoinUrl(address: string) {
+  const successBackLink = encodeURI(`https://explorer.stacks.co/address/${address}?chain=mainnet`);
+  const thirdPartyLink = encodeURI('https://hiro.so');
+
+  return `https://www.okcoin.com/bridge/thirdBridge?isThirdBridge=1&partnerId=10002&crypto=STX&address=${address}&thirdPartyName=Hiro&fiatAmount=100&currency=USD&successBackLink=${successBackLink}&thirdPartyLink=${thirdPartyLink}`;
+}
+
+function makeTransakUrl(address: string) {
+  const apiKey = isProduction ? TRANSAK_API_KEY_PRODUCTION : TRANSAK_API_KEY_STAGING;
+  const subdomain = isProduction ? 'global' : 'staging-global';
+  const screenTitle = 'Buy Stacks';
+
+  return `https://${subdomain}.transak.com?apiKey=${apiKey}&cryptoCurrencyCode=STX&exchangeScreenTitle=${encodeURI(
+    screenTitle
+  )}&defaultPaymentMethod=credit_debit_card&walletAddress=${address}`;
+}
+
+export function getProviderUrl(address: string, provider: string) {
+  switch (provider) {
+    case ActiveFiatProviders.Binance:
+      return '';
```

Comment:
Note: These are placeholders for now. I need a url to the faq page. I can simplify this once I have it bc they will all be going to the same faq page (just sending provider and address as params).